### PR TITLE
Josephina  mobile correct answer discussion

### PR DESCRIPTION
**Pull request by josephinaim**
screen S.1.7 and S.1.9 has correct answer displayed with index and check mark icon

### CODE REVIEW

File: mobile/ios/Podfile.lock [hunk 1]
```diff
@@ -387,6 +387,8 @@ PODS:
     - React-Core
   - RNCMaskedView (0.1.11):
     - React
+  - RNDeviceInfo (10.0.2):
+    - React-Core
```
Comment (maniramezan) [inline mobile/ios/Podfile.lock:391]: This has been removed from the codebase, the `pod install` shouldn't cause this to be added. Do you have the latest from `dev`?

File: mobile/src/Student/screens/Game/PhaseTwoBasicGamePlay/index.js [hunk 2]
```diff
@@ -23,6 +23,11 @@ import HorizontalPageView from "../../../components/HorizontalPageView"
 import ScrollableQuestion from "../Components/ScrollableQuestion"
 import AnswerOptionsPhaseTwo from "./AnswerOptionsPhaseTwo"
 import HintsView from "../Components/HintsView"
+import RoundTextIcon from "../../../components/RoundTextIcon"
+
+const indexToLetter = (index) => {
+    return String.fromCharCode(65 + index)
```
Comment (maniramezan) [inline mobile/src/Student/screens/Game/PhaseTwoBasicGamePlay/index.js:29]: Nice find, if you want to go this path, some comments on the code would be useful for future you and others reading the code.

**Review by maniramezan — CHANGES_REQUESTED** [review]

File: mobile/ios/Podfile.lock (hunk 1 above)
Comment (josephinaim) [reply mobile/ios/Podfile.lock:391]: I do, but for some reason, pod install still causes these changes?

File: mobile/src/Student/screens/Game/PhaseTwoBasicGamePlay/index.js (hunk 2 above)
Comment (josephinaim) [reply mobile/src/Student/screens/Game/PhaseTwoBasicGamePlay/index.js:29]: Sounds good! Will add some commenting.

**Pull request closed without merging**